action.yml

```diff
@@ -10,26 +10,38 @@ inputs:
   config:
     description: 'Path to config file'
     required: false
+  format:
+    description: 'Output format. Allowed values: sarif, plain, json. Default: "plain"'
+    required: false
+    default: 'plain'
   output-file:
-    description: 'Filename to store output.  Default "kubelinter.log"'
+    description: 'Filename to store output. File will be overwritten if it exists. Default: "kubelinter.log"'
     required: false
     default: 'kubelinter.log'
   version:
-    description: 'Version of kube-linter to use. Default "latest"'
+    description: 'Version of kube-linter to use. E.g. "0.2.4". Default: "latest"'
     required: false
     default: 'latest'
 runs:
   using: "composite"
   steps:
     - name: Download kube-linter
+      shell: bash
       run: |
-        set -euo pipefail
+        set -u
         case "${{ runner.os }}" in
           macOS)   OS=darwin ;;
           Windows) OS=windows ;;
           *)       OS=linux ;;
         esac
-        RELEASE_INFO=$(curl --silent --show-error --fail https://api.github.com/repos/stackrox/kube-linter/releases/${{ inputs.version }})
+        RELEASE_URL='https://api.github.com/repos/stackrox/kube-linter/releases/latest'
+        if [ "${{ inputs.version }}" != "latest" ]; then
+          RELEASE_URL='https://api.github.com/repos/stackrox/kube-linter/releases/tags/${{ inputs.version }}'
+        fi
+        RELEASE_INFO=$(curl --silent --show-error --fail \
+          --header 'authorization: Bearer ${{ github.token }}' \
+          --header 'Cache-Control: no-cache, must-revalidate' \
```

Comment:
Maybe add a comment about why this is necessary?

Reply:
Added.